library/src/main/java/xmtpv3.kt

```diff
@@ -47,17 +47,14 @@ import kotlinx.coroutines.suspendCancellableCoroutine
 open class RustBuffer : Structure() {
     // Note: `capacity` and `len` are actually `ULong` values, but JVM only supports signed values.
     // When dealing with these fields, make sure to call `toULong()`.
-    @JvmField
-    var capacity: Long = 0
-    @JvmField
-    var len: Long = 0
-    @JvmField
-    var data: Pointer? = null
+    @JvmField var capacity: Long = 0
```

Comment:
Can you run `command + option + L` on this file to format it. That should bring down the changes.

Reply:
version file added here 247d07cf8483792097f2017ea9e7f7431ba1e706